packages/@sanity/desk-tool/src/DeskTool.test.tsx

```diff
@@ -436,91 +445,11 @@ describe('DeskTool', () => {
           },
           type: 'document',
         })
-      })
-    })
-
-    it('passes the correct arguments to the structure child/items', async () => {
```

Comment:
note: i deleted this test because it already has some weird outputs and i've fix those in #2916.